rubocop-shopify.gemspec

```diff
@@ -3,7 +3,7 @@
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.name        = "rubocop-shopify"
-  s.version     = "2.17.0"
+  s.version     = "2.17.1"
```

Comment:
I figured a patch version bump is adequate since we're just addressing a cop rename warning.